Gems/ROS2/Code/Include/ROS2/Sensor/ROS2SensorComponent.h

```diff
@@ -54,5 +63,13 @@ namespace ROS2
         virtual void Visualise(){};
 
         float m_timeElapsedSinceLastTick = 0.0f;
+        double m_timeElapsedSinceLastFrequencyTick = 0.0f;
```

Comment:
Please separate this out to a class that ROS2SensorComponent does not depend on (it is an addition).